docs/howto/operate/manual-deploy.md

```diff
@@ -0,0 +1,73 @@
+# Manual deploy a hub
+
+The [`deploy.py`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py) script is a cmd utility built to help deploy one or all the hubs to their Kubernetes clusters, but  also to build the user image. Checkout the [](../configure/update-env.md) section on how to use the deploy script to update the user environment.
+
+## Deploy script arguments and usage
+Running:
+```
+python3 deploy.py deploy --help
+```
+Will provide information about the `deploy.py` deploy options:
+
+```bash
+usage: deploy.py deploy [-h] [--skip-hub-health-test]
+                        [cluster_name] [hub_name]
+
+positional arguments:
+  cluster_name
+  hub_name
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --skip-hub-health-test
+
+```
+
+* Running the script without arguments
+
+    Deploys all cluster and hubs in the `hubs.yaml` file.
+    ```bash
+    python3 deploy.py deploy
+    ```
+* `cluster_name` argument
+
+    Deploys all the hubs in the cluster passed as argument (the hubs listed in `hubs.yaml` file unde `<cluster>`)
+    ```bash
+    python3 deploy.py deploy <cluster>
+    ```
+* `hub_name` argument
+
+    Deploys only the hub passed as argument. The cluster must match the hub's cluster listed in the `hubs.yaml`.
+    ```bash
+    python3 deploy.py deploy <cluster> <hub>
+    ```
+* `--skip-hub-health-test` flag
+
+    Skips the default health checks ran by the deploy script. Checkout [testing the hub section](test-hub.md) to learn more about the hub health tests.
+    ```bash
+    python3 deploy.py deploy --skip-hub-health-test
+    ```
+
+### Pre-requisites
+
+Before using any of these `deploy.py` commands, you first need to make sure you are authenticated in all the right places and have the right credentials exported as environmental variables. This is because the deploy script need to be able to authenticate into the given cluster and have access to the unique `proxy.secretToken` needed to deploy the proxy pod. It also needs to know how to connect to the Auth0 account in order to determine and configure each hub's auth provider.
+
+```{note}
+To enable the deploy script to authenticate into the cluster, checkout the [Fetch Cluster credentials](cmd-access.md) section.
```

Comment:
You only need to be authenticated to the two-eye-two-see project, since we have direct credentials for everything else under `secrets/`. They're encrypted with [sops](https://github.com/mozilla/sops), and the key is in the two-eye-two-see project.